### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by jeffzh4ng**
## Purpose
Updates our PR template to prompt the dev to list out possible approaches of the solution.

image

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 _Describe the problem or feature in addition to a link to the issues._
 
 ## Approach
-_How does this change address the problem?_
```
Comment (xujustinj) [inline .github/pull_request_template.md:5]: while we're at it maybe we can add a section for FE screenshots

Comment (lesliexin) [reply .github/pull_request_template.md:5]: +1

**Review by xujustinj — APPROVED** [review]
In the future we can probably fine-tune the order in which the headings come up

**Comment by jeffzh4ng** [comment]
Damn it I forgot to change the PR title to include emoji.

**Pull request merged**